lib/test/pack.cpp

```diff
@@ -11,17 +11,42 @@
 
 namespace {
 
+void check_size(const char* fmt, const int* src_v_size, const int* dst_v_size){
```

Comment:
check_size shouldn't be a separate function, and/or should not look like this with pointer shenanigans. _Intent_